### PR TITLE
Change location of binaries folder

### PR DESCRIPTION
**Pull request by WojcikMike**
Connects to Particular/PlatformDevelopment#723

Moving binaries 1 level up in a hierachy
The reason for that is to allign with others repositories where binaries is on the same level as src folder

### CODE REVIEW

**Comment by WojcikMike** [comment]
@Particular/azure-storage-queues-maintainers please review and merge

**Comment by melkio** [comment]
All "azure" related repos have the same convention. Do we have to update all of them?

**Comment by WojcikMike** [comment]
@andreasohlund told me that it is probably better to update as keeping binaries in the src directory doesn't make sense. But as I am not maintainer I am not in the position to say, that is the only way to do it.

**Comment by melkio** [comment]
My question was only to know if we have to create a specific issue for each azure related repos. That's all :)

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 1]
```diff
@@ -24,23 +24,23 @@
...
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\binaries\</OutputPath>
+    <OutputPath>..\..\binaries\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DocumentationFile>..\binaries\NServiceBus.Azure.Transports.WindowsAzureStorageQueues.xml</DocumentationFile>
+    <DocumentationFile>..\..\binaries\NServiceBus.Azure.Transports.Win  dowsAzureStorageQueues.xml</DocumentationFile>
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: why the extra spaces?

Comment (WojcikMike) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: Thanks for catching that @SimonCropp fixed

**Comment by WojcikMike** [comment]
@melkio yes I think we should.

**Comment by Scooletz** [comment]
@melkio My understanding is that @Particular/azure-storage-queues-maintainers will support this, but applying in other repos is their's maintainers responsibility.

**Comment by WojcikMike** [comment]
@melkio I created Issues in:
* AzureStoragePersistence
* AzureServiceBus
* AzureBlobStorageDataBus
* AzureCloudService

Let me know if I missed something

**Comment by SeanFeldman** [comment]
:+1:

**Pull request merged**